docs/SUMMARY.md

```diff
@@ -1,18 +1,24 @@
 # Summary
 
-* [Introduction](README.md)
-* [Getting Started](getting-started/README.md)
-* [Installation](installation/README.md)
-  * [Windows Installer](installation/windows.md)
-* [Running nio](running-nio/README.md)
-  * [In the Cloud](running-nio/in-the-cloud.md)
+* Introduction
+
+* [Welcome](README.md)
+* [Install nio](installation/README.md)
+  * [On Windows](installation/windows.md)
+  * [Supported Systems](installation/supported-systems.md)
+* [Run nio](running-nio/README.md)
+  * [In a nio-Managed Cloud](running-nio/in-the-cloud.md)
   * [Locally](running-nio/locally.md)
-  * [Using Docker](running-nio/docker.md)
-  * [Long Running Instances](running-nio/long-running.md)
+  * [In a Docker Container](running-nio/docker.md)
+  * [For a Long Time](running-nio/long-running.md)
```

Comment:
Maybe "For Long Durations"

Or is there a better phrase we could use? It's really more like "how to really run nio in any kind of real setting"

Reply:
I thought about "In Production"...